akka-http-core/src/main/scala/akka/http/scaladsl/model/MediaType.scala

```diff
@@ -336,6 +336,7 @@ object MediaTypes extends ObjectRegistry[(String, String), MediaType] {
   val `application/javascript`                                                    = awoc("javascript", "js")
   val `application/json`                                                          = awfc("json", HttpCharsets.`UTF-8`, "json")
   val `application/json-patch+json`                                               = awfc("json-patch+json", HttpCharsets.`UTF-8`)
+  val `application/merge-patch+json`                                              = awfc("merge-patch+json", HttpCharsets.`UTF-8`)
```

Comment:
please mirror this in javadsl

Reply:
Hi @ktoso I have added it to the JavaDSL, apologies for missing that :-).